packages/syntax-core/src/SelectList/SelectList.tsx

```diff
@@ -20,6 +20,7 @@ const SelectList = ({
   errorText,
   helperText,
   label,
+  hideLabel = false,
```

Comment:
@nathan-ong instead of a boolean prop, how do you feel about:

```tsx
labelDisplay: "visible" | "hidden"
```

Reply:
ahh, ok, yeah that makes sense to me!